packages/cli/src/commands/init.ts

```diff
@@ -145,15 +148,26 @@ export const initCommand = async (options: InitCommandOptions) => {
 
   await addConfigurationToPackageJson(resolvedPath, resolvedOptions);
 
-  await printNextSteps(resolvedOptions, authorizedKey);
+  await printNextSteps(resolvedOptions, authorizedKey, nextJsVersion, nextJsDir);
   telemetryClient.init.completed(resolvedOptions);
 };
 
-async function printNextSteps(options: ResolvedOptions, authorizedKey: WhoamiResponse) {
+async function printNextSteps(
+  options: ResolvedOptions,
+  authorizedKey: WhoamiResponse,
+  nextJsVersion: number,
+  nextJsDir: "pages" | "app"
+) {
   const projectUrl = `${options.triggerUrl}/orgs/${authorizedKey.organization.slug}/projects/${authorizedKey.project.slug}`;
 
   logger.success(`✅ Successfully initialized Trigger.dev!`);
 
+  if (nextJsDir === "pages") {
+    const message = `Detected you are using Next.js Page Router version ${nextJsVersion}, check out this article on our docs on how to complete the setup. https://trigger.dev/docs/documentation/quickstarts/nextjs`;
```

Comment:
I'm not sure what this is meant to be for? Is this for doing the transpile module stuff? It doesn't look like we actually have anything in our docs referencing how to finish setting up the Page Router version

Reply:
I updated the docs in the first commit to show how to do that